dnsdist-1.9.yaml

```diff
@@ -0,0 +1,199 @@
+      - busybox
+      - cargo-auditable
+      - cmake
+      - fstrm-dev
+      - gnutls-dev
+      - libbpf-dev # eBPF/XDP support
+      - libcap-dev
+      - libedit-dev
+      - libquiche
+      - libsodium-dev
+      - libtool
+      - lmdb-dev
+      - lua${{vars.lua-version}}-dev
+      - luajit-dev
+      - mtdev-dev
+      - net-snmp-dev
+      - nghttp2-dev
+      - openssl-dev
+      - pkgconf-dev
+      - protobuf-dev
+      - py3-yaml
+      - python-3.12
+      - quiche
+      - ragel-dev
+      - re2-dev
+      - systemd
+      - systemd-dev
+      - wslay-dev
+      - xdp-tools-dev
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/PowerDNS/pdns
+      tag: dnsdist-${{package.version}}
+      expected-commit: a1ab4e4d0764d2f36eec35f169f81e30281a9b09
+
+  - working-directory: ./pdns/dnsdistdist
+    pipeline:
+      - uses: autoconf/configure
+        with:
+          opts: |
+            --sysconfdir=/etc/dnsdist \
+            --sbindir=/usr/bin \
+            --enable-dnscrypt \
+            --enable-dns-over-tls \
+            --enable-dns-over-https \
+            --enable-dns-over-quic \
+            --enable-dns-over-http3 \
+            --enable-unit-tests \
+            --enable-yaml \
+            --enable-systemd \
+            --enable-dnstap \
+            --with-libsodium \
+            --with-net-snmp \
+            --with-lua=lua${{vars.lua-version}} \
+            --with-gnutls \
+            --with-h2o \
+            --with-libcap \
+            --with-nghttp2 \
+            --with-re2
+      - uses: autoconf/make
+      - uses: autoconf/make-install
+      - runs: make check
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Compatibility package for dnsdist
+    dependencies:
+      provides:
+        - dnsdist-compat=${{package.full-version}}
+      runtime:
+        - python-${{vars.py-version}}
+        - py${{vars.py-version}}-jinja2
+        - tini
+        - libcap-utils
+        - coreutils # For env -S support
+        - merged-usrsbin
+        - wolfi-baselayout
+    pipeline:
+      - working-directory: ./dockerdata
+        pipeline:
+          - runs: |
+              # https://github.com/PowerDNS/pdns/blob/master/Dockerfile-dnsdist
+              mkdir -p ${{targets.contextdir}}/etc/dnsdist
+              mkdir -p ${{targets.contextdir}}/usr/local/bin/
+              mkdir -p ${{targets.contextdir}}/etc/dnsdist/conf.d
+              mkdir -p ${{targets.contextdir}}/etc/dnsdist/templates.d
+              install -Dm755 startup.py ${{targets.contextdir}}/usr/local/bin/dnsdist-startup
+              install -Dm755 dnsdist.conf ${{targets.contextdir}}/etc/dnsdist/dnsdist.conf
+              install -Dm644 dnsdist-resolver.lua ${{targets.contextdir}}/etc/dnsdist/dnsdist-resolver.lua
+              ln -sf /usr/bin/dnsdist ${{targets.contextdir}}/usr/local/bin/dnsdist
+    test:
+      pipeline:
+        - runs: |
```

Comment:
Perhaps this 
```
        - name: "Verify the symlink"
          uses: test/tw/symlink-check
 ```

Reply:
Tested this but it failed:
```
    test:
      environment:
        contents:
          packages:
            - ${{package.name}} # symlink-check expects the binary to be installed
      pipeline:
        - uses: test/tw/symlink-check
```

```
2025/08/08 00:07:21 WARN + symlink-check '--allow-absolute=false' '--allow-dangling=false' '--packages=dnsdist-1.9-compat'
2025/08/08 00:07:21 INFO INFO[symlink-check]: Tested [1] symlinks with [symlink-check]. [0/1] passed.
2025/08/08 00:07:21 INFO
2025/08/08 00:07:21 INFO FAILED SYMLINKS:
2025/08/08 00:07:21 INFO FAIL[symlink-check]: absolute symlink: /usr/local/bin/dnsdist -> /usr/bin/dnsdist
```

There was a known bug or smth with `symlink-check` as its not stable yet, i'll add this check later.